src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
```

Comment:
> It's members are not consider reachable either way as soon as the finalizer is done

Unless it's resurrected.  I'm not sure if that's what @danmosemsft is concerned about.